src/lib_json/json_value.cpp

```diff
@@ -254,6 +254,15 @@ Value::CZString::CZString(const CZString& other)
   storage_.length_ = other.storage_.length_;
 }
 
+#if JSON_HAS_RVALUE_REFERENCES
+Value::CZString::CZString(CZString&& other)
+  : cstr_(other.cstr_),
```

Comment:
Google style would be to indent this more like:
```c++
Value::CZString::CZString(CZString&& src)
    : cstr_(src.cstr_),
      index_(src.index_) {
  src.cstr_ = nullptr;
}
```

(And join cstr_ and index_ onto a single line actually)